src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/MSBuildAPIUtility.cs

```diff
@@ -60,7 +56,7 @@ private static Project GetProject(string projectCSProjPath, IDictionary<string,
             var projectRootElement = TryOpenProjectRootElement(projectCSProjPath);
             if (projectCSProjPath == null)
             {
-                throw new Exception(Strings.Error_MsBuildUnableToOpenProject);
+                throw new Exception(string.Format(CultureInfo.CurrentCulture, Strings.Error_MsBuildUnableToOpenProject, projectCSProjPath));
```

Comment:
I would also make these InvalidOperationExceptions, it is only slightly more useful, but it at least could be caught without catching ALL exceptions.